keras_retinanet/preprocessing/generator.py

```diff
@@ -75,27 +75,33 @@ def load_annotations(self, image_index):
         raise NotImplementedError('load_annotations method not implemented')
 
     def load_annotations_group(self, group):
-        annotations_group = [self.load_annotations(image_index) for image_index in group]
+        return [self.load_annotations(image_index) for image_index in group]
 
+    def check_images_annotations(self, image_group, annotations_group, group):
```

Comment:
`check` implies that the function returns true/false. However, it filters the annotations. How about `filter_annotations` or similar?

Reply:
Makes sense, I agree.